desktop/src/main/java/bisq/desktop/main/funds/withdrawal/WithdrawalView.java

```diff
@@ -330,8 +331,9 @@ private void onWithdraw() {
                     feeEstimationTransaction = walletService.getFeeEstimationTransactionForMultipleAddresses(fromAddresses, sendersAmount);
                 }
                 checkNotNull(feeEstimationTransaction, "feeEstimationTransaction must not be null");
-                Coin fee = feeEstimationTransaction.getFee();
-                sendersAmount = feeExcluded ? amountAsCoin.add(fee) : amountAsCoin;
+                Coin dust = getDust(feeEstimationTransaction);
+                Coin fee = feeEstimationTransaction.getFee().add(dust);
+                sendersAmount = feeExcluded ? amountAsCoin.add(fee) : amountAsCoin.add(dust);
```

Comment:
The added complexity of the dust on top of the fee makes this code really hard to read. It's not transparent to neither the developer nor the user what will happen here. The dust amount is so low that it's unlikely to cause monetary damage, but users expect a certain amount to be sent and when the dust is added it might cause confusion. (If I got it right,the dust is added as fee or sent amount depending on whether feeExcluded is set or not) 

I suggest making this code easier to follow, and adding a notice to the user that dust might be padded to the fee and make sure it's only padding the fee.

Reply:
:+1: @sqrrm your suggestions sound great, I am working on implementing them!

Reply:
> adding a notice to the user that dust might be padded to the fee

I propose the following message in cases where dust is detected (shown in the following screenshot at the top of the popup):

![image](https://user-images.githubusercontent.com/47253594/77838615-39813680-713b-11ea-96e5-9c651911a0e9.png)

Reply:
@m52go could you have a look at the user facing language. We're trying to avoid confusion and I feel I might be too deep here to have a good judgement of what a normal user will find helpful and what's just adding to extra confusion.

Reply:
Seems straightforward to me overall.

2 small things.

* [satoshi](https://github.com/bisq-network/bisq/pull/4093/files#diff-ff484a282dd668b4e76956ad0b761971R352) should be plural. It's already pluralized below in the 'required transaction fee' line

* might be nice to change the 2 "transaction fee" mentions to "mining fee" to be more specific and match the text on the 'send funds' screen

Reply:
@m52go @sqrrm your changes have been implemented.

![image](https://user-images.githubusercontent.com/47253594/77980063-eee2f400-72cb-11ea-8864-5c583f6664cf.png)